lib/ansible/modules/net_tools/nios/nios_host_record.py

```diff
@@ -60,10 +60,14 @@
         description:
           - Configures the IPv4 address for the host record. Users can dynamically
             allocate ipv4 address to host record by passing dictionary containing,
-            I(nios_next_ip) and I(CIDR network range). See example
+            I(nios_next_ip) and I(CIDR network range). If user wants to add or
+            remove the ipv4 address from existing record, I(address_add/address_remove)
+            aliases need to be used. See examples
```

Comment:
Okay, I'm confused. Are these aliases, or do they have different functionality depending on which is used?

Reply:
Yea, these are aliases having different functionality depending upon their usage, if user want to create host record they'll use `address`, and if they have to add/remove IP address from existing host then they can use `address_add/address_remove`.